packages/atlas/src/views/viewer/ChannelView/ChannelSearch.styles.ts

```diff
@@ -1,57 +1,24 @@
 import styled from '@emotion/styled'
 
-import { Button } from '@/components/_buttons/Button'
 import { Input } from '@/components/_inputs/Input'
-import { cVar, media, sizes, transitions } from '@/styles'
+import { media, sizes } from '@/styles'
 
 type TextFieldProps = {
   isOpen?: boolean
   isSearching?: boolean
 }
 export const StyledInput = styled(Input)<TextFieldProps>`
-  transition: all ${transitions.timings.regular} ${transitions.easing};
-  will-change: max-width;
-  align-items: center;
-  position: relative;
-
-  ${media.sm} {
-    max-width: ${({ isOpen }) => (isOpen ? '200px' : '0px')};
-  }
-
-  input {
-    height: 40px;
-    padding: 10px 16px 10px 42px;
-    caret-color: ${cVar('colorCoreBlue500')};
-    font: ${cVar('typographyDesktopT200')};
-    letter-spacing: ${cVar('typographyDesktopT200LetterSpacing')};
-    text-transform: ${cVar('typographyDesktopT200TextTransform')};
-    border-radius: 0;
-
-    ${media.sm} {
-      ${({ isOpen }) => isOpen === false && 'box-shadow: none !important'};
-    }
-
-    :focus {
-      box-shadow: 0 0 0 1px ${cVar('colorCoreNeutral50')};
-    }
-
-    ::-webkit-search-cancel-button,
-    &[type='search'] {
-      /* stylelint-disable-next-line property-no-vendor-prefix */
-      -webkit-appearance: none;
-    }
-  }
-`
+  width: 100%;
 
-export const SearchButton = styled(Button)`
-  position: absolute;
+  /* to align it with SearchIcon button */
+  margin-left: -${sizes(1)};
```

Comment:
We should probably apply this only on desktop:
![Screenshot 2022-06-27 at 09 29 50](https://user-images.githubusercontent.com/12646744/175884218-41d804d4-edd5-4c0b-9081-32705438c467.jpg)

Reply:
fixed